### PR TITLE
Codeless GDevelop Game Jam

### PR DESCRIPTION
**Pull request by edmundlzen**
Thanks for making a PR! Please fill in the below:

Name: Edmund

Do you run a club?: No.

What relevant experiences do you have to help you write this Jam?: Made a game using GDevelop before, decent coding experience.

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?: Yes.

### CODE REVIEW

**Comment by edmundlzen** [comment]
No idea if a codeless game jam would fly, but I felt like it would lower the barrier of entry of making a game for a lot of people!

File: jams/codeless-gd-game-jam.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Q: What platforms will be supported (i.e. MacOS, Windows, Chromebook, Mobile, Browser, etc)?
+
+A: Everything except mobile would probably work fine with GDevelop's web editor.
+
+---
+
+Q: How will you allow Club Leaders to add their own project to the Jam presentation (giving them a sense of ownership over the meeting)?
+
+A: A showcase session at the end, letting people demo their own versions of the game.
```
Comment (Arpan-206) [inline jams/codeless-gd-game-jam.md]: I am not sure if I'm satisfied with this answer, can't point out why but a look at the closed PRs may be helpful.

File: jams/codeless-gd-game-jam.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Q: What Club Members will walk away with (both in terms of knowledge and in terms of product)
+
+A: A cool game that they can customize easily and a basic understanding on the high level logic behind game development.
+
+---
+
+Q: What makes this workshop fun or interesting for Club Members?
+
+A: Making their own game!
```
Comment (Arpan-206) [inline jams/codeless-gd-game-jam.md]: Can you expand this a bit?

File: jams/codeless-gd-game-jam.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- Set up: Download a GDevelop project with some starter art and code. Sign up for an account over at GDevelop and open the editor.
+1. Basic introduction to GDevelop, it's UI and how it's event blocks work.
+2. Add event blocks to implement spawning of cats (allies).
+3. Add event blocks for a HP system and enemy and allied bases.
+4. Add event blocks for spawning of enemies.
+5. Exporting for different platforms.
+6. Customization time! Change up the art, voicelines to make this game truly yours!
+- Profit, you have now got your own game that you can share with anyone and customize for your enjoyment! Replace your friend's face and voice for the enemy?
+
+# Q & A
```
Comment (Arpan-206) [inline jams/codeless-gd-game-jam.md:28]: More verbose answers for this section will be very helpful.

File: jams/codeless-gd-game-jam.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+_Requirements:_ Any computer that can run a browser.
+
+_Language used:_ Event blocks similar to scratch, but much more powerful
+
+_Customization opportunity:_ Customize art of the enemies? Put some weird voice lines, add more heroes with different attacks, add enemies that have different special abilities? Your imagination is the limit!
+
+_Platform Limitations:_ Browser/computer based game development.
+
+_Time estimation:_ 1-2 hours
```
Comment (Arpan-206) [inline jams/codeless-gd-game-jam.md]: That's a pretty big range, it should be around 10 minutes max, like 50-60 mins or something.

File: jams/codeless-gd-game-jam.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+### Breakdown
```
Comment (Arpan-206) [inline jams/codeless-gd-game-jam.md:16]: Mind adding time estimates for each step?

**Review by Arpan-206 — CHANGES_REQUESTED** [review]
Answers to the Q/A are a little vague and short, time estimates for each step.

File: jams/codeless-gd-game-jam.md (hunk 5 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md:16]: Done

File: jams/codeless-gd-game-jam.md (hunk 4 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md]: Changed the time estimation to be more logical

File: jams/codeless-gd-game-jam.md (hunk 1 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md]: Hmm 🤔 For this question specifically, a showcase session at the end seems to be about the only method possible for this kind of thing?

File: jams/codeless-gd-game-jam.md (hunk 2 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md]: Done

File: jams/codeless-gd-game-jam.md (hunk 3 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md:28]: Made the answers for some sections more verbose

Comment (Arpan-206) [reply jams/codeless-gd-game-jam.md:28]: I was talking about answers to all the Q/A questions, they all can do with a little more explanation.

File: jams/codeless-gd-game-jam.md (hunk 1 above)
Comment (Arpan-206) [reply jams/codeless-gd-game-jam.md]: @sarthaktexas what do you think?

File: jams/codeless-gd-game-jam.md (hunk 3 above)
Comment (edmundlzen) [reply jams/codeless-gd-game-jam.md:28]: Done, added some more details to the answer as best as I could.

**Comment by SerenityUX** [comment]
This is awesome!!! What do you think Hack Clubbers would learn through making a game with this no-code tool? I love that you're pushing the bounds of what we initially imagined a Jam looking like.

**Comment by edmundlzen** [comment]
@SerenityUX Thanks! I think Hack Clubbers would learn more about the logic behind games without diving into code first and lets them learn how to solve problems , it's sort of like what scratch teaches kids, but more advanced.

**Comment by sarthaktexas** [comment]
howdy @edmundlzen ! we thank you for your proposal, but it looks like it doesn't fit the standard we're looking for in our jams. i'm sorry to say, because of that, we can't accept your jam proposal.

**Pull request closed without merging**